src/main.ts

```diff
@@ -263,23 +263,26 @@ export async function run(): Promise<void> {
 
   const pathsOutput = paths.join(separator)
 
+  const hasCustomPatterns = (
+      files !== '' ||
+      filesFromSourceFile !== '' ||
+      excludedFiles !== '' ||
+      excludedFilesFromSourceFile !== ''
+  )
+
   if (pathsOutput) {
     const pathsOutputFile = await tempfile('.txt')
     await fs.writeFile(pathsOutputFile, pathsOutput, {flag: 'w'})
 
     core.setOutput('paths-output-file', pathsOutputFile)
     core.saveState('paths-output-file', pathsOutputFile)
     core.info(`Successfully created paths-output-file: ${pathsOutputFile}`)
+  } else if (hasCustomPatterns) {
+    core.warning("No paths found using the specified patterns")
```

Comment:
🚫 **[eslint]** <[prettier/prettier](https://github.com/prettier/eslint-plugin-prettier#options)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Replace `"No·paths·found·using·the·specified·patterns"` with `'No·paths·found·using·the·specified·patterns'`
```suggestion
    core.warning('No paths found using the specified patterns')
```